packages/dev/core/src/Rendering/IBLShadows/iblShadowsVoxelRenderer.ts

```diff
@@ -564,25 +563,32 @@ export class _IblShadowsVoxelRenderer {
 
     private _renderVoxelGrid() {
         if (this._voxelizationInProgress) {
-            const allRTsReady = this._renderTargets.every((rt) => rt.isReadyForRendering());
+            let allRTsReady = true;
+            for (let i = 0; i < this._renderTargets.length; i++) {
+                const rttReady = this._renderTargets[i].isReadyForRendering();
+                allRTsReady &&= rttReady;
+            }

```

Comment:
I did an update lately in this code (see https://github.com/BabylonJS/Babylon.js/compare/master...Popov72:Babylon.js:fix-ibl-shadows?expand=1), else it would crash in my latest testing.

Reply:
Thanks! I brought this in.